doc/src/modules/solvers/solveset.rst

```diff
@@ -398,7 +398,7 @@ How does ``solveset`` ensure that it is not returning any wrong solution?
     >>> from sympy import symbols, S, pprint, solveset
     >>> x, n = symbols('x, n')
     >>> pprint(solveset(abs(x) - n, x, domain=S.Reals), use_unicode=True)
-    [0, ∞) ∩ {n}
+    {x | x ∊ {-n, n} ∧ ({{n} ∈ [0, ∞)})}
```

Comment:
Actually, I did this is by design.
For example, if we have more than one condition to be satisfied as in this case [[Link](https://github.com/sympy/sympy/pull/10751/files#diff-85baa04bbf4e1dfd9128782738e45424R132)],
I think this representation will be better for handling various conditions.
What do you think ?